en/docs/guides/login/logging-into-salesforce-using-is.md

```diff
@@ -0,0 +1,287 @@
+    <tr class="odd">
+    <td>Request Signature Method</td>
+    <td>RSA-SHA1</td>
+    </tr>
+    <tr class="even">
+    <td>Assertion Decryption Certificate</td>
+    <td>Assertion not encrypted</td>
+    </tr>
+    <tr class="odd">
+    <td>SAML Identity Type</td>
+    <td><p>Assertion contains user's salesforce username</p></td>
+    </tr>
+    <tr class="even">
+    <td>SAML Identity Location</td>
+    <td><p>Identity is in the NameIdentifier element of the Subject statement</p></td>
+    </tr>
+    <tr class="odd">
+    <td>Service Provider Initiated Request Binding</td>
+    <td>HTTP POST</td>
+    </tr>
+    <tr class="even">
+    <td>Identity Provider Login URL</td>
+    <td><code>                               https://localhost:9443/samlsso                             </code></td>
+    </tr>
+    <tr class="odd">
+    <td>Custom Logout URL</td>
+    <td><code>                               https://localhost:9443/samlsso                             </code></td>
+    </tr>
+    <tr class="even">
+    <td>Custom Error URL</td>
+    <td>Leave blank</td>
+    </tr>
+    <tr class="odd">
+    <td>User Provisioning Enabled</td>
+    <td>Leave blank</td>
+    </tr>
+    </tbody>
+    </table>
+
+    Click **Save** to save your configurations.
+
+9.  Navigate to **Company Settings** in the left navigation pane and click **My Domain**.
+    
+10. Click **Deploy to Users**. Click **Ok** to the confirmation message
+    that appears.
+    
+11. In the page that appears, you must configure the **Authentication Configuration** section. Scroll down to this section and click
+    **Edit**.
+    
+12. Under **Authentication Service**, select **SSO** instead of **Login
+    Page**.  
+    ![authentication-service-sso](../../assets/img/guides/auth-config-salesforce.png)
+    
+13. Click **Save**.
+
+
+## Configure email address as the username
+
+For details on configuring WSO2 Identity Server to use an email address as the username, refer [Using Email Address as the Username](insert-link).
+
+## Configure the service provider
+
+Follow the steps given below to configure salesforce as a service
+provider in WSO2 IS.
+
+1.  Sign in. Enter your username and password to log on to the
+    [Developer Portal](https://localhost:9443/developer-portal).
+      
+    You can login using the user name, **admin@wso2.com** and password, **admin** for the first time. 
+    
+2.  Select **Applications** from the left hand side panel. 
+
+3. Click on **New Application**.
+
+    ![add-sp-salesforce.png](../../assets/img/guides/add-sp-salesforce.png)
+
+4. Click on **SAML Web Application**.
+    
+3.  Fill in the **Name** and optionally, provide a brief
+    **Description** of the application.  
+
+    ![service-provider-name](../../assets/img/guides/create-saml-app-salesforce.png)
+    
+4.  Click **Next**.
+
+5. Enter the **issuer ID** as `https://saml.salesforce.com`. 
+
+6. Enter the **assertion consumer URL**. This can be obtained from the Salesforce end. 
```

Comment:
assertion consumer URL --> Assertion consumer URL

from the Salesforce end --> from Salesforce